mods/d2k/rules/infantry.yaml

```diff
@@ -200,13 +200,13 @@ sardaukar:
 	Inherits@AUTOTARGET: ^AutoTargetGroundAssaultMove
 	Buildable:
 		Queue: Infantry
-		BuildPaletteOrder: 80
-		Prerequisites: ~barracks.harkonnen, upgrade.barracks, high_tech_factory, ~techlevel.medium
+		BuildPaletteOrder: 50
+		Prerequisites: ~palace.sardaukar, ~techlevel.high
 		BuildDuration: 81
 		BuildDurationModifier: 40
 		Description: Elite assault infantry\n  Strong vs Infantry, Vehicles\n  Weak vs Artillery
```

Comment:
Could you add something to the description to tell them apart? E.g. `Corrino elite assault ...`